resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml

```diff
@@ -1623,6 +1623,7 @@ spec:
       labels:
         app: eventing-controller
         eventing.knative.dev/release: "v0.10.0"
+        kyma-project.io/dashboard: "event-mesh"
```

Comment:
(nit) none of those quotes are required in YAML

Reply:
Right, I changed in all the values.